R/util.R

```diff
@@ -115,10 +115,22 @@ uses_unary_minus <- function(expr) {
 }
 
 
+## These would do well to cope with things like
+## expr_plus(quote(i - 1), 1)
+## expr_plus(quote(i + 2), 3)
 expr_minus <- function(a, b) {
   if (is.numeric(a) && is.numeric(b)) {
     a - b
   } else {
     call("-", a, b)
   }
 }
+
+
+expr_plus <- function(a, b) {
+  if (is.numeric(a) && is.numeric(b)) {
+    a + b
```

Comment:
Code coverage - this numeric add doesn't get covered - will try and suggest something below...